scripts/update-projects-version.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/env python3
```

Comment:
A new script should include Copyright header and some description on what it does, how it works.